client/state/happychat/middleware.js

```diff
@@ -193,6 +262,27 @@ export const sendAnalyticsLogEvent = ( connection, { getState }, { meta: { analy
 	} );
 };
 
+export const sendActionLogsAndEvents = ( connection, { getState }, action ) => {
+	const state = getState();
+
+	// If there's not an active Happychat session, do nothing
+	if ( ! isHappychatClientConnected( state ) || ! isHappychatChatAssigned( state ) ) {
+		return false;
+	}
+
+	// If there's analytics metadata attached to this action, send analytics events
+	if ( has( action, 'meta.analytics' ) ) {
+		sendAnalyticsLogEvent( connection, action );
```

Comment:
Yes, this is intentional. A single action may have both analytics data attached to it, and may also be an action type we want to send to the operator. For example:

```
{
    type: 'HAPPYCHAT_BLUR',  // This is the action type, which we want to send to operator
    meta: {
        analytics: [ ... ]  // This action also has analytics data attached, which we may be interested in
    }
}
```